preppal-fe/src/app.tsx

```diff
@@ -1,8 +1,7 @@
 import 'bootstrap/dist/css/bootstrap.min.css';
 import './app.css';
 import { BrowserRouter, Routes, Route } from "react-router-dom";
-
-import NavBar from './components/navbar/navbar';
+import NavBar from './components/NavBar/NavBar';
```

Comment:
import change does not reflect file name.

Reply:
fixed all naming conventions